frontend/src/components/dashboard/visitors/ButtonComponent.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <div
+        class="button-card bg-white d-inline-flex justify-content-start align-items-center"
+    >
+        <div
+            class="card-image"
+        >
+            <img
+                class="button-icon"
+                :src="iconSrs"
```

Comment:
to make it work with actual icon name change this row to
```suggestion
                :src="require(`@/assets/icons/${iconKey}.png`)"
```